src/ExpandableDataPicker/ExpandableDataPicker.js

```diff
@@ -70,6 +70,7 @@ module.exports = kind(
 	*/
 	collectionChanged: function () {
 		this.$.list.set('collection', this.collection);
+		if (this.collection && !this.$.list.hasNode()) this.$.list.refresh();
```

Comment:
The only alternative I can come up with would be to iterate over the collection and find the first entry with `active: true`. That'd be more performant but it does make the pretty key assumption that   that property will be used to trigger the `onActivate` event. If a dev used a custom property or a custom control (other than CheckboxItem), it wouldn't work as expected.

At a minimum, we'd need to document that constraint or look for a more comprehensive change.